warnings in amqp-cpp that are treated as errors in our build

hello.

we are using AMQP-CPP library in our project and having troubles building it, because several headers have warnings in them, and they are treated as errors:

at
```
envelope.h line 68
envelope.h line 125
libevent.h line 67
```

```
C:\...\.conan\data\amqp-cpp\4.3.11\_\_\package\8cf01e2f50fcd6b63525e70584df0326550364e1\include\amqpcpp/envelope.h(68): warning C4244: 'argument': conversion from 'uint64_t' to 'uint32_t', possible loss of data 
...\include\amqpcpp/envelope.h(125): warning C4244: 'argument': conversion from 'const uint64_t' to 'uint32_t', possible loss of data (compiling source file ...) 
...\include\amqpcpp/libevent.h(67): warning C4244: 'argument': conversion from 'intptr_t' to 'int', possible loss of data (compiling source file ..) 
```


----
we believe it is a good practice to treat warnings as errors.
could you please fix those?

--- a/include/amqpcpp/voidfield.h
+++ b/include/amqpcpp/voidfield.h
@@ -33,7 +33,7 @@ public:
      *  Construct based on incoming data
      *  @param  frame
      */
-    VoidField(InBuffer &frame) {}
+    VoidField(InBuffer &frame) { (void)frame; }
 
     /**
      *  Destructor
@@ -65,7 +65,7 @@ public:
      *  Write encoded payload to the given buffer.
      *  @param  buffer      OutBuffer to write to
      */
-    virtual void fill(OutBuffer &buffer) const override {}
+    virtual void fill(OutBuffer &buffer) const override { (void)buffer; }
 
     /**
      *  Get the type ID that is used to identify this type of

